package.json

```diff
@@ -32,21 +32,25 @@
     "url": "https://github.com/CrowdStrike/eslint-config-crowdstrike-node/issues"
   },
   "homepage": "https://github.com/CrowdStrike/eslint-config-crowdstrike-node#readme",
+  "engines": {
+    "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
+  },
   "devDependencies": {
     "@crowdstrike/commitlint": "^7.0.0",
-    "eslint": "^8.0.0",
+    "eslint": "^8.23.0",
     "eslint-config-crowdstrike": "10.1.0",
     "eslint-config-crowdstrike-node": "link:",
     "eslint-plugin-json-files": "^4.0.0",
-    "eslint-plugin-n": "^14.0.0",
+    "eslint-plugin-n": "^17.9.0",
     "remark-cli": "^12.0.0",
     "remark-preset-lint-crowdstrike": "^4.0.0",
     "renovate-config-standard": "^2.0.0",
     "standard-version": "^9.0.0"
   },
   "peerDependencies": {
+    "eslint": ">=8.23.0",
     "eslint-config-crowdstrike": ">=1",
     "eslint-plugin-json-files": ">=0.8",
-    "eslint-plugin-n": ">=14"
+    "eslint-plugin-n": ">=17"
```

Comment:
So we will release a new major version of this packages with these changes, so its reasonable to set these as required peerdeps yeah?

Reply:
yep

Reply:
We should make the peer dep minimum the lowest possible, so whenever https://github.com/eslint-community/eslint-plugin-n/issues/80 was fixed.

Reply:
for example https://github.com/kellyselden/eslint-config-sane-node/commit/52fe1eb691b66ed391659afd136accb86bc12cac

Reply:
For my own edification, are we ok then with the breaking changes between `15.1` and `17`? Or does it work differently than im understanding

Reply:
our version of eslint-plugin-n is a devDep. The peerDep means consumers control their own version of eslint-plugin-n. We just set the min version to 15.1. If they want to bump to 17, that's on them.

Reply:
Roger, i guess i was confused because we are using `n/recommended` in `index.js` so i assumed we are exporting that ruleset in our package, maybe its actually a `dependency` not a `devDep/peerDep`? I'm still grokking how these rulesets cascade, thanks for explaining

Reply:
we are using n/recommended of whatever the consumer version supplies.

Reply:
for example https://github.com/CrowdStrike/commitlint/blob/v8.0.1/package.json#L62